builds/any/rootfs/stretch/common/overlay/etc/systemd/system/networking.service.d/switchdev-online.conf

```diff
@@ -1,2 +1,3 @@
 [Unit]
-After=network-pre.target
+After=local-fs.target network-pre.target
```

Comment:
Instead of `local-fs.target` maybe use `sysinit.target` ([bootup(7)](https://www.freedesktop.org/software/systemd/man/bootup.html)), and remove the [`DefaultDependencies=no`](https://www.freedesktop.org/software/systemd/man/systemd.unit.html#DefaultDependencies=)?